### PR TITLE
Birmingham Class 2 - Elisha Mesquitta - Git&GitHub - Week 2

### PR DESCRIPTION
**Pull request by lish-dev**
Your Name: Elisha Mesquitta
Your City: Birmingham
Your Slack Name: Lisha

# Homework Details

- Module: Git& Github
- Week: 2

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -39,75 +43,91 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 It send it to a friend
+Clone is another term for 'paste'.
+
 
 ## Question 11
 
 What is a Pull Request?
 
 When you send a file over the internet
+Pull request is another term for asking others for feedback.
```
Comment (jip1984) [inline homework.md:84]: Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.

File: homework.md [hunk 2]
```diff
@@ -5,30 +5,34 @@
...
+What is the sum 2 * 3 * 6 * 1 * 10 * 34?
 1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
 Hyper Tall Massive Language
+Hyper Text Markup Language
```
Comment (jip1984) [inline homework.md:29]: Hypertext is one word

File: homework.md (hunk 1 above)
Comment (lish-dev) [reply homework.md:84]: Noted, thanks Jon 👍

File: homework.md (hunk 2 above)
Comment (lish-dev) [reply homework.md:29]: Will amend in due course thanks Jon.

File: homework.md [hunk 3]
```diff
@@ -39,75 +43,91 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-When people want to show off code to each other they can put it on Github
+1. Tracks changes 2.Instant feedback on collaborative work. 3.Version control
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
 You delete it
+Fork is another term for 'copy'.
```
Comment (ironkamosi) [inline homework.md:68]: Hi Elisha,

The answer is partially correct. Can you expand on your answer and describe the process.

All the best,

Aaron

**Pull request closed without merging**